container/container_windows.go

```diff
@@ -10,6 +10,10 @@ import (
 	containertypes "github.com/docker/docker/api/types/container"
 )
 
+const (
+	containerSecretMountPath = `C:\ProgramData\Docker\secrets`
```

Comment:
I actually wonder if this should follow the `graphdir`, so that setting a custom graphdir would also change the location for these; https://github.com/docker/docker/blob/master/daemon/config_windows.go#L13